src/pages/Home.jsx

```diff
@@ -1,11 +1,14 @@
 import GamesContainer from '@components/GamesContainer/GamesContainer';
 import Layout from '@components/Layout/Layout';
+import { useState } from 'react';
 
 export default function Home() {
+    const [singleColumn, setSingleColumn] = useState(false);
+
     return (
         <>
-            <Layout />
-            <GamesContainer />
+            <Layout singleColumn={singleColumn} setSingleColumn={setSingleColumn} />
```

Comment:
I think there's an issue with this change to the layout. The purpose of the layout is having the side bar and header, as well as any component rendered by the outlet. With this change, you are always displaying the outlet and the games container below it, right?

Reply:
I did not change the components positioning.
Home being desplayed below layout was already happening in main.
And adding the sidebar at the left of the games container is something that will not happen, unless with add proper changes.

I think it is a weird way of arranging the parts of the page adding the side bar inside the Layout.

In my oppinion a cleaner arrangement would be:

<              header and title section             >
<div><side bar><games container></div>

This can be easely achieved by the Home component and the Last Searches component

Also, having a parent component that encapsulates the layout and games container, allows me to define the "singleColumn" state, in a way that it can be passed down to its children (the games container, and the columns toggle)

@elios11

Reply:
Well but the previous design allows having the header and sidebar always on the Layout component pages and just point any other routes into it, making it more re-usable. You could still pass state down with context maybe? I just don't think repeating the calling of components when we already have a HOC is the best option here.
And why wouldn't we add the sidebar to the side of the games container? That's what we need to actually do, right? Like having a container wrapping the outlet and the sidebar with a display flex would just achieve that immediatly.
Maybe I'm wrong though, I wish I could test your changes, but for some reason this is the only branch that I can't access from my local repository. Will keep trying, though.

Reply:
Yes, adding the sidebar to the side of the games container is what we have to do.
I just don't understand how we would achieve it without adding additional divs.

We know the GamesContainer needs to be diplayed below the Header. 
Therefore the GamesContainer needs to be displayed below the Layout.

if you add the Sidebar inside the Layout. Won't the GamesContainer be displayed below it?

@elios11

Reply:
Oh, yeah. But we can just add a wrapper to those and apply a display: flex and that's it, right? Just adjust the gap, I think it's okay to use divs to adjust the layout of different components, so we don't have to apply the layout stylings directly into the components, and they are independant from those styles.